<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:"Aptos Display";}
@font-face
	{font-family:HellasTimes;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0in;
	font-size:12.0pt;
	font-family:"Arial",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:.25in;
	margin-right:0in;
	margin-bottom:4.0pt;p
	margin-left:0in;
	page-break-after:avoid;
	font-size:20.0pt;
	font-family:"Aptos Display",sans-serif;
	color:#0F4761;
	font-weight:normal;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:8.0pt;
	margin-right:0in;
	margin-bottom:4.0pt;
	margin-left:0in;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Aptos Display",sans-serif;
	color:#0F4761;
	font-weight:normal;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-top:8.0pt;
	margin-right:0in;
	margin-bottom:4.0pt;
	margin-left:0in;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Arial",sans-serif;
	color:#0F4761;
	font-weight:normal;}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{mso-style-link:"Title Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:4.0pt;
	margin-left:0in;
	font-size:28.0pt;
	font-family:"Aptos Display",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpFirst, li.MsoTitleCxSpFirst, div.MsoTitleCxSpFirst
	{mso-style-link:"Title Char";
	margin:0in;
	font-size:28.0pt;
	font-family:"Aptos Display",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpMiddle, li.MsoTitleCxSpMiddle, div.MsoTitleCxSpMiddle
	{mso-style-link:"Title Char";
	margin:0in;
	font-size:28.0pt;
	font-family:"Aptos Display",sans-serif;
	letter-spacing:-.5pt;}
p.MsoTitleCxSpLast, li.MsoTitleCxSpLast, div.MsoTitleCxSpLast
	{mso-style-link:"Title Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:4.0pt;
	margin-left:0in;
	font-size:28.0pt;
	font-family:"Aptos Display",sans-serif;
	letter-spacing:-.5pt;}
p.MsoBodyText, li.MsoBodyText, div.MsoBodyText
	{mso-style-link:"Body Text Char";
	margin:0in;
	font-size:14.0pt;
	font-family:"HellasTimes",serif;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Aptos Display",sans-serif;
	color:#0F4761;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Aptos Display",sans-serif;
	color:#0F4761;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Times New Roman",serif;
	color:#0F4761;}
span.TitleChar
	{mso-style-name:"Title Char";
	mso-style-link:Title;
	font-family:"Aptos Display",sans-serif;
	letter-spacing:-.5pt;}
span.BodyTextChar
	{mso-style-name:"Body Text Char";
	mso-style-link:"Body Text";
	font-family:"HellasTimes",serif;}
.MsoChpDefault
	{font-family:"Aptos",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:115%;}
@page WordSection1
	{size:595.3pt 841.9pt;
	margin:1.0in 1.25in 1.0in 1.25in;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US style='word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoTitleCxSpFirst><span lang=EL style='font-size:18.0pt;color:black'>&nbsp;</span></p>

<p class=MsoTitleCxSpMiddle><span lang=EL style='color:black'>&nbsp;</span></p>

<p class=MsoTitleCxSpMiddle><span lang=EL style='color:black'>&nbsp;</span></p>

<p class=MsoTitleCxSpMiddle><span lang=EL style='font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><span
style='font-size:22.0pt;font-family:"Times New Roman",serif;color:black'>SEQUENCING
OF dsDNA</span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><span
style='font-size:18.0pt;font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><span
style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><b><span
style='font-family:"Times New Roman",serif;color:black'>8<sup>th</sup>-12<sup>th</sup>
of November 2002</span></b></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><span
style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><b><span
style='font-size:20.0pt;font-family:"Times New Roman",serif;color:black'>Assistants:</span></b></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><b><span
style='font-size:20.0pt;font-family:"Times New Roman",serif;color:black'>Pavel
Pavlo</span></b></p>

<p class=MsoTitleCxSpMiddle align=center style='text-align:center'><b><span
style='font-size:20.0pt;font-family:"Times New Roman",serif;color:black'>Shashi
Bushan</span></b></p>

<p class=MsoTitleCxSpMiddle><b><span style='font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></b></p>

<p class=MsoTitleCxSpMiddle><b><span style='font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></b></p>

<p class=MsoTitleCxSpMiddle><b><span style='font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></b></p>

<p class=MsoTitleCxSpMiddle><b><span style='font-family:"Times New Roman",serif;
color:black'>Name: Evangelos Papadopoulos.</span></b></p>

<p class=MsoTitleCxSpMiddle><b><span style='font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></b></p>

<p class=MsoTitleCxSpLast><span style='color:black'>INTRODUCTION</span></p>

<p class=MsoBodyText><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></p>

<p class=MsoBodyText><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:black'>����������� DNA sequencing is a fundamental method used in biology
research to help us identify and record the base sequence and composition of
the actual nucleic acids of living organisms or artificially synthesized DNA.</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>����������� The
method we used in this lab (dideoxy method) was developed by F. Sanger. It
makes use of the mechanism of DNA synthesis by DNA polymerizers. The DNA to be
sequenced must be first denatured to single strand DNA. After that an
appropriate primer is used. This is a short DNA fragment that is known to be
complementary to one position in the original DNA. The primer binds to its
appropriate site and then the DNA polymerization starts in the presence of DNA
polymerizer and the four deoxynucleosides, dATP, dCTP, dTTP, dGTP.</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>����������� But
a small amount of ddATP, ddCTP, ddTTP and ddGTP is added to four different
samples. The composition of� dideoxy ddNTP (N= A, C, T, G) does not allow
further continuation of the polymerization reaction. Therefore the reaction
stops. The exact position where the reaction will stop is a random event for
every molecule but since in each tube we have only one type of ddNTP we know in
what type of base the reaction will stop.</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>����������� In
this way the molecular weight of the synthesized DNA fragments is an indicator
of the positions of the according bases. After that we can use any kind of
technique, like electrophoresis to determine the order of the different
molecular weights or sizes synthesized and therefore sequence the gene.</span></p>

<p class=MsoNormal><span style='font-size:14.0pt;font-family:"HellasTimes",serif;
color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:14.0pt;font-family:"HellasTimes",serif;
color:black'>&nbsp;</span></p>

<h1><span style='font-size:28.0pt;color:black'>MATERIAL AND METHODS</span></h1>

<p class=MsoNormal align=center style='text-align:center'><b><u><span
style='font-size:18.0pt;font-family:"HellasTimes",serif;color:black'><span
 style='text-decoration:none'>&nbsp;</span></span></u></b></p>

<p class=MsoBodyText><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:black'>����������� We were provided with the DNA sample and the primers.
I used the SK primer. Then we proceeded according to the experimental procedure
that is described in the laboratory notes. </span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>First step is the
Denaturing of ds DNA.</span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>This is accomplished by the addition of 6M NaOH.</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>Then annealing of the
primer to the DNA.</span></p>

<p class=MsoNormal style='margin-left:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>Just by adding the primer and a buffer in the appropriate
conditions.</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>After we have the
denatured DNA with the primer bound on it, it is ready for beginning the
polymerization reaction. We prepare 4 different tubes containing the
corresponding mixtures for termination of the polymerization in A, C, G and T. </span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>4.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>We add polymerase and
radioactive dNTP for labeling to the DNA+primer mixture. And we leave it for
sometime so as the polymerization reaction starts taking place.</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>5.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>Then we transfer a
portion of the above polymerizing mixture into each of the 4 base terminating
tubes. The reaction is therefore terminated but in different position in each
tube. Particularly in the A tube it will terminate in an A base etc.</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>6.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>We prepared the
electrophoresis polyacrilamide gel and we let it polymerize for one day.</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>7.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>We loaded the samples
in the gel and run the electrophoresis for two hours.</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>8.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>We took the gel out in
a paper washed and dry it.</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>9.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>We used a radiation
scanner to view the results.</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<h1><span style='color:black'>RESULTS</span></h1>

<p class=MsoNormal align=center style='text-align:center'><b><u><span
style='font-size:14.0pt;font-family:"HellasTimes",serif;color:black'><span
 style='text-decoration:none'>&nbsp;</span></span></u></b></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>����������� The
sample we used for some reason did not work out well. The quality of the
scanned gel image was very poor and although we could see the base lines their
position was not clear enough to allow sequencing. Therefor we were given
another gel image to analyze. (See: <u>Page I</u> )</span></p>

<h2><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></h2>

<h2><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:black'>����������� In <u>page II</u> is a printout of the gel image and
the sequence that was derived from that. I used this sequence consisting of 55
base nucleotides to try and locate the gene from which they come.</span></h2>

<h2><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></h2>

<h2><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:black'>����������� We used the internet database of NCBI to identify the
gene. The results of the query are in <u>pages� II-IV</u>. As you can see it
has an almost perfect match with the genome of Azotobacter vinelandii. There
were only 2 gaps. <u>Page I</u>. The first gap is a G base which although it is
not present we can see a blank row in the gel. So it was something missing
there. And the second gap is in a position where there should be 4 C-bases but
we detect only three. Probably because of some base loop that the strands form
and/or from not perfect denaturing.</span></h2>

<h2 style='text-indent:.5in'><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:black'>In particular this base sequence is located in a protein encoding
gene for a dimmeric 2Fe-2S protein that is not essential for aerobic nitrogen
fixation but confers oxygen protection to the nitrogenase in vitro and in vivo.
An essential task as the time of half-life of nitrogenases in the presence of
Oxygen vary from 30 sec to a few minutes.� </span></h2>

<h2 style='text-indent:.5in'><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:black'>The base sequence encoding this protein is in <u>Pages V-VI. </u>And
you can see highlighted the sequenced portion. Which is reverse and
complementary to the gel sequence.</span></h2>

<h2><span style='color:black'>�</span></h2>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center'><b><u><span
style='font-size:18.0pt;font-family:"HellasTimes",serif;color:black'>COMMENTS</span></u></b></p>

<p class=MsoNormal><span style='font-size:14.0pt;font-family:"HellasTimes",serif;
color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:14.0pt;font-family:"HellasTimes",serif;
color:black'>&nbsp;</span></p>

<h3><span style='color:black'>Answering questions</span></h3>

<p class=MsoNormal><span style='font-size:14.0pt;font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:14.0pt;font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>1)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>Why is it necessary to
pre-run the gel before loading the sample?</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>The electrophoresis gel was pre-run half an hour before loading
the </span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>DNA
sequencing sample. This is done for heating up the gel before so as to be in a
temperature equilibrium. Like preheating an oven. The diffusion process is
temperature dependant and an uneven distribution of the temperature throughout
the sample would lead to different electrophoretic results in various positions
in the gel. </span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>����������� The
same is valid for other chemical component gradients that have to be in an
equilibrium state for the electrophoresis to work reliably.</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>����������� Moreover,
we had some release of extra urea from the saturated gel. This had to be washed
out before loading the sample or otherwise might block the pathway for
initializing electrophoresis.</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>2)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>Why do we read the
sequence from the bottom of the gel and not from the top?</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>The sequencing is accomplished by differentiating the size or/and
molecular weight of the different produced DNA fragments. Therefore, if for
example an A base is close to the primer and another one is farther away from
the primer we will have in the A terminating tube a short fragment of copied
DNA ending in A and a long fragment of DNA ending in A too. In electrophoresis
the smaller and lighter molecules propagate faster and so cross a bigger
distance while for the bigger and heavier molecules propagation is more
difficult and they remain closer to the original position. </span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>We conclude that a small DNA fragment ending in A means an A base
is closer to the primer and we will find it more away from the original
position (Top) and closer to the bottom. So, we have to start reading from
bottom to the top. </span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>It is also important to note that the gel gives the sequence
complementary to the template where primer binds.</span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>3)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>You cloned your gene of
interest, which is approximately 1000bp into a vector. Now you want to sequence
that gene in order to find out the nucleotide sequence. After the first
sequencing, you have successfully sequenced 512bp. How will you find the rest of
the sequence? (two alternatives)</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>We can find the rest of the sequence depending on the position of
the already sequenced fragment in our gene of interest and our knowledge of the
vector.� Generally, we can continue by using primers derived from our first
sequencing or use primers appropriate for the cloning vector.</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<span style='font-size:12.0pt;font-family:"Times New Roman",serif;color:black'><br
clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>Let�s say for example that the first 512 bp we have sequenced are
the first in the 3�-5� direction then we have another 488 bp. We can chemically
synthesize another primer complementary to a few of the last sequenced bp. By
using this primer, we can follow the same experimental technique as in our
practical and continue sequencing of the gene in the 3�-5� direction. We can
also continue sequencing in the opposite direction if we use a primer equal to
the first few bp of our gene. Because this primer will bind to the
complementary strand and continue sequencing in the reverse direction.</span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>However, this technique has the disadvantage that we should
chemically construct a primer after our first sequencing. But if we know the
sequence of our vector a few base pairs around our gene insertion point we
could have already prepared appropriate primers to proceed sequencing both
directions downstream and upstream. So by 512bp sequencing we would have easily
reconstructed the whole 1000bp long gene with some possible overlapping in the
middle for confirmation and correct spacing of the two sequences.</span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>����������� </span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:.25in;text-indent:-.25in'><span
style='font-family:"Times New Roman",serif;color:black'>4)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span><span
style='font-family:"Times New Roman",serif;color:black'>Define cDNA. Which are
the advantages of sequencing cDNA?</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='font-family:"Times New Roman",serif;
color:black'>cDNA is the DNA synthesized from an mRNA template by the use of
reverse transcriptases. These are enzymes originally located in retrovirus and
they serve the inverse transcription required for them to synthesize DNA so as
to exploit the invading cell translational system. cDNA is an important tool in
genome research since it allows the sequencing of a particular mRNA coding for
a particular protein. So, it helps discriminate between introns and exons
(expressed and not expressed sequences) in the original DNA.</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-family:"HellasTimes",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-family:"HellasTimes",serif;color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

</body>

</html>
